helm/cosmo/charts/router/templates/config-map.yaml

```diff
@@ -12,7 +12,7 @@ data:
   config.yaml: |-
     # User-supplied common configuration:
     {{- if .Values.commonConfiguration }}
-    {{- .Values.commonConfiguration | nindent 4 }}
+    {{- tpl .Values.commonConfiguration . | nindent 4 }}
```

Comment:
Could you document this behaviour in the chart comments of `commonConfiguration ` ? After running `make docs`, the docs are regenerated.